docker-compose.yml

```diff
@@ -24,9 +24,19 @@ services:
             - '/usr/src/app/node_modules'
         ports:
             - 3008:8080
-        depends_on: 
+        depends_on:
             - unitednepali-server
-    
+
+    unitednepali-documentation:
+        build:
+            context: packages/documentation
+            dockerfile: Dockerfile
+        ports:
+          - 3009:3000
+          - 35729:35729
```

Comment:
no point exposing the livereload port because you'll have to rebuild the container, if you made changes to the src files in documentation

Reply:
I thought that's why